src/webportal/src/app/job/job-view/fabric/job-detail/conn.js

```diff
@@ -100,6 +100,27 @@ export async function fetchSshInfo() {
   }
 }
 
+export function getTensorBoardUrl(jobInfo, rawJobConfig) {
+  let port = null;
+  let ip = null;
+  if (rawJobConfig.hasOwnProperty('extras') && rawJobConfig.extras.hasOwnProperty('tensorBoardStr')) {
+    const randomStr = rawJobConfig.extras.tensorBoardStr;
+    const tensorBoardStr = `TensorBoard_${randomStr}`;
+    const tensorBoardPortStr = `tensorBoardPort_${randomStr}`;
+    const obj = jobInfo.taskRoles;
+    if (obj.hasOwnProperty(tensorBoardStr)) {
+      if (obj[tensorBoardStr].taskStatuses[0].taskState == 'RUNNING') {
```

Comment:
please use [`===`](https://stackoverflow.com/a/523647)

Reply:
Got it. Thanks.